options.go

```diff
@@ -3,12 +3,19 @@ package mongods
 import "time"
 
 var (
-	defaultConfig = config{opTimeout: 5 * time.Second, txnTimeout: 30 * time.Second}
+	defaultConfig = config{
+		opTimeout:  5 * time.Second,
+		txnTimeout: 30 * time.Second,
+		dbName:     "godsmongo",
```

Comment:
"god's mongo"

Reply:
😄 
Will go away for the safer "specifying dbname is mandatory to avoid some disaster".